tutorials/sphinx_tuto/tensordict_shapes.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+"""
+Manipulating the shape of a TensorDict
+======================================
+**Author**: `Vincent Moens <https://github.com/vmoens>`_
+
+In this tutorial you will learn how to manipulate the shape of a ``TensorDict`` and its
+contents.
+"""
+
+##############################################################################
+# When we create a ``TensorDict`` we specify a ``batch_size``, which must agree with
+# the leading dimensions of all entries in the ``TensorDict``. Since we have a guarantee
+# that all entries share those dimensions in common, ``TensorDict`` is able to expose
+# a number of methods with which we can manipulate the shape of the ``TensorDict`` and
+# its contents.
+
+# sphinx_gallery_start_ignore
+import warnings
+
+warnings.filterwarnings("ignore")
+# sphinx_gallery_end_ignore
+import torch
+from tensordict.tensordict import TensorDict
+
+##############################################################################
+# Reshaping a ``TensorDict``
+# --------------------------
+#
+# ``TensorDict.reshape`` works just like ``torch.Tensor.reshape``. It applies to all of
```

Comment:
We can use `:func:ˋ for functions, `:class:` for classes and `:mod:` for modules. If the path is complete this will generate a link to the api reference in the rendered tutorial.